pkg/apis/networking/types.go

```diff
@@ -158,6 +158,9 @@ type NetworkPolicy struct {
 	// Priority represents the relative priority of this Network Policy as compared to
 	// other Network Policies. Priority will be unset (nil) for K8s Network Policy.
 	Priority *float64
+	// TierPriority represents the priority of the Tier associated with this Network
+	// Policy.
+	TierPriority *uint32
```

Comment:
Can we add a comment here saying TierPriority will never be nil? i.e. if tier is spec is "" it will be assigned TierApplication

Reply:
this will be nil for K8s NetworkPolicy.. at least as of now.. we don't collect them in Application Tier

Reply:
Yes I realized that, updated the comment

Reply:
makes sense .. will update